packages/ui/.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
         project: ['./tsconfig.eslint.json'],
         tsconfigRootDir: __dirname,
     },
-    rules: {},
+    rules: { 'func-style': 'off' },
```

Comment:
Let's not disable this rule for the whole package. You can disable the rule for specific line, and that is okay in your use-case (forwardRef)